### ISSUE TITLE
Share checkpoints and scripts to run the trained model

### ISSUE THREAD

**Issue by Obikate** [CandleLabAI/PCBSegClassNet#3]
I would really like to apply your model on PCB images, but I can see that the necessary checkpoints and scripts for that are not in the repository. Could you please add them?

**Comment by betogulliver** [comment]
I've successfully trained the model using the FPIC dataset (loss curve LGTM)

could you please share a simple script to do inference for a single image?

I've tried doing the following:

pred = model.predict(im_array)

the shape/dtype of the pred is
(1, 512, 512, 26) float32

512, 512 = im_size
26 = num_classes + bg

are these 'pred' results the logits of the prediction? 
how can I convert them into a colored mask images?

thanks again sharing your work, much appreciated!

**Comment by PladsElsker** [comment]
@betogulliver Do you still have the dataset? Can you share it on hugging face if the license allows it? The FPIC link provided in the repo is dead and the authors are unresponsive. 

Edit:
Nevermind found it here
https://physicaldb.ece.ufl.edu/index.php/fics-pcb-image-collection-fpic/

Edit 2:
Nevermind nevermind, there are only 271 images in the dataset I linked, so it seems to only be a sample.

**Comment by ArsalanJamali** [comment]
@PladsElsker Hi, 

I followed your given link to download the dataset but it asks for account creation which I did but my account never gets created. I only get a message that you will receive an email with login info, which i never receive. 

Could you please share the dataset in some way.?

**Comment by PladsElsker** [comment]
@ArsalanJamali I think I still have it somewhere on my drive. If I can find it, I'll share it as a hugging face dataset here later tonight 👍

Just keep in mind it's only a sample dataset, I was not able to find the full dataset.

**Comment by PladsElsker** [comment]
@ArsalanJamali Unfortunately, 2 things:
1) The dataset is gated because it is protected under copyright, therefore I do not have the authorization to redistribute it, and I apparently risk legal actions against me if I do share it publicly
2) My version of the dataset does not contain the labels, so it's probably useless anyway

The best I can do is share the official link to the dataset, which I already did. 
If it doesn't work, maybe try to find another dataset or make your own. 

Good luck!
